demo/EXPLOR3R/auto-drive.py

```diff
@@ -48,6 +48,8 @@
 ir = InfraredSensor(); assert ir.connected
 ts = TouchSensor();    assert ts.connected
 
+print('Robot Starting')
```

Comment:
Since this is a demo, we could use this as an opportunity to demonstrate espeak functionality, and change the line to
```py
Sound.speak('Robot starting').wait()
```
It will also be easier to notice when launched from brickman interface.